launcher/ui/dialogs/MSALoginDialog.ui

```diff
@@ -49,14 +49,25 @@ aaaaa</string>
     </widget>
    </item>
    <item>
-    <widget class="QDialogButtonBox" name="buttonBox">
-     <property name="orientation">
-      <enum>Qt::Horizontal</enum>
-     </property>
-     <property name="standardButtons">
-      <set>QDialogButtonBox::Cancel</set>
-     </property>
-    </widget>
+    <layout class="QHBoxLayout" name="horizontalLayout">
+     <item>
+      <widget class="QPushButton" name="actionButton">
+       <property name="text">
+        <string>Action button placeholder. bbbbb</string>
```

Comment:
bbbbbbb

Reply:
This is a reference to the label placeholder a few lines above :sweat_smile: 
https://github.com/PolyMC/PolyMC/blob/35caa3c21afed378c73e2a448ac92e356f502281/launcher/ui/dialogs/MSALoginDialog.ui#L26-L28

Reply:
Now that I think about it, I don't really know if this was intended by the original author of the placeholder :O

Reply:
It doesn't need a placeholder though, the text never changes.

Reply:
I wasn't sure how translations are handled with .ui files. Can I just put the text there and expect it to be replaced (if a translation exists)?